pkgs/applications/window-managers/i3/bumblebee-status/plugins.nix

```diff
@@ -0,0 +1,377 @@
+{ pkgs
+, python3Packages
+, ...
+}:
+with pkgs;
+with python3Packages; let
+  mkPluginCfg =
+    { name
+    , requires ? [ ]
+    , requiresUnpackaged ? [ ]
+    , optional ? [ ]
+    , warnings ? [ ]
+    ,
+    }: rec {
+      inherit name requires requiresUnpackaged optional;
+      warnings =
+        let
+          warn = dep: ''Plugin "${name}": Dependency `${dep}` is not known to be contained in Nixpkgs, so plugin might fail at runtime.'';
+          requiresUnpackagedWarnings = map warn requiresUnpackaged;
+        in
+        warnings ++ requiresUnpackagedWarnings;
```

Comment:
I am not sure where you're going with this. It seems that you commented all plugins that would trigger a warning? So now this code is somehow dead. Maybe add a comment explaining the situation?

Reply:
Replaced dead code with a comment.